src/utils/combineReducers.js

```diff
@@ -97,9 +98,16 @@ export default function combineReducers(reducers) {
     }
   });
 
+  if (sanityError) {
+    console.error(sanityError.message);
+  }
+
```

Comment:
Yes, we previously `console.error`'d only in development, too.
I think we should hide this `console.error` behind the dev flag as well.

Of course, if it's not helpful, it's better to remove it. I'm not sure whether the confusion from error printed twice is bigger than the help from a proper stack trace. Thoughts?